datastore/datastore.go

```diff
@@ -91,6 +147,26 @@ func init() {
 	boltdb.Register()
 }
 
+// DefaultScopes returns a map of default scopes and it's config for clients to use.
+func DefaultScopes(dataDir string) map[string]*ScopeCfg {
```

Comment:
It seems odd to see a get API changing a package global data.
Since the dataDir is something that cannot change after init, Why is it that this is not setup once and just use it in DefaultScopes function without having to pass the dataDir as a parameter ?